src/components/Scaler/index.js

```diff
@@ -271,11 +302,13 @@ class Scaler extends Component {
           subDomainsByItemId[itemId][axis] || newSubDomain;
         const existingSpan = existingSubDomain[1] - existingSubDomain[0];
 
-        const limits = ((domainsById || {})[itemId] || {})[axis] ||
-          (axis === String(Axes.time)
-            ? // FIXME: Phase out this single timeDomain thing.
-              this.props.dataContext.timeDomain
-            : undefined) || [Number.MIN_SAFE_INTEGER, Number.MAX_SAFE_INTEGER];
+        const limits = stripPlaceholderDomain(
```

Comment:
Why we need to strip out placeholder here?

Reply:
If a domain wasn't specified, it will be given a placeholder value (so that it's a normal array downstream). If this is the case, we want to strip it so that we can go back to the [MIN, MAX] array.

You could argue that instead of using PLACEHOLDER_DOMAIN as the default, maybe we should just use the [MIN, MAX] array, but then it becomes less-obvious that it's a placeholder.